### PR TITLE
fix(graphql): fix `graphql.operation.name` field

### PR DESCRIPTION
**Pull request by dchambers**
## Which problem is this PR solving?

- https://github.com/open-telemetry/opentelemetry-js-contrib/issues/705
- https://github.com/open-telemetry/opentelemetry-js-contrib/issues/847

## Short description of the changes

- Change `graphql.operation.name` to send the GraphQL [operation-name](https://graphql.org/learn/queries/#operation-name) (a critical field for GraphQL o11y) instead of the GraphQL operation-type.
- Continue to send the GraphQL operation-type as `graphql.operation.type`, since this is still useful for segmenting read queries from write mutations, etc.

## Checklist

- [x] Ran `npm run test-all-versions` for the edited package(s) on the latest commit if applicable.

### CODE REVIEW

**Comment by dchambers** [comment]
@dyladan, might you be able to help move this PR out of its current state?

File: plugins/node/opentelemetry-instrumentation-graphql/src/enums/AttributeNames.ts
Comment (dyladan) [inline plugins/node/opentelemetry-instrumentation-graphql/src/enums/AttributeNames.ts:22]: Consider using the semantic conventions package?

**Review by dyladan — APPROVED** [review]

**Review by vmarchaud — APPROVED** [review]

**Comment by dchambers** [comment]
Hi @obecny 👋. You seem to be the last reviewer on the roster for this PR. Any chance you'd be able to take a look please?

**Comment by dchambers** [comment]
Hi @obecny, it's been another week. Any chance you'd be able to look at this PR please?

**Comment by dyladan** [comment]
@obecny is no longer a maintainer on the project. sorry for the confusion

**Comment by Flarna** [comment]
he is still part of the approver group and [the owner](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/bf39b908fd64cec92c862b2deca5d760ddcf4509/.github/component_owners.yml#L28-L29) of the graphql instrumentation.

**Comment by dchambers** [comment]
> @obecny is no longer a maintainer on the project. sorry for the confusion

I see, thank you @dyladan. Do you know who would now be the right person to ping regarding getting this PR merged in that case?

**Comment by dchambers** [comment]
> he is still part of the approver group and [the owner](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/bf39b908fd64cec92c862b2deca5d760ddcf4509/.github/component_owners.yml#L28-L29) of the graphql instrumentation.

This makes sense too @Flarna 👍 , because he's showing as the last person I need an approval from before I can merge.

**Review by Flarna — APPROVED** [review]

**Comment by dyladan** [comment]
I am the right person :)

**Comment by dchambers** [comment]
Thank you guys! 🙏

**Pull request merged**
